app.go

```diff
@@ -390,6 +390,52 @@ func (app *App) GetAllRecords(fields []string) ([]*Record, error) {
 	}
 }
 
+func isAllowedLang(allowedLangs []string, lang string) bool {
+	for _, allowedLang := range allowedLangs {
+		if lang == allowedLang {
+			return true
+		}
+	}
+	return false
+}
+
+// GetProcess retrieves the process management settings
+// This method can only be used when App is initialized with password authentication
+// lang must be one of default, en, zh, ja, user
+func (app *App) GetProcess(lang string) (process *Process, err error) {
+	type request_body struct {
+		App  uint64 `json:"app,string"`
+		lang string `json:"lang,string"`
```

Comment:
This `lang` field has json tag but is private thus will not be included in the request body.